core/src/main/java/hudson/cli/LogoutCommand.java

```diff
@@ -27,7 +30,13 @@ protected void printUsageSummary(PrintStream stderr) {
     @Override
     protected int run() throws Exception {
         ClientAuthenticationCache store = new ClientAuthenticationCache(checkChannel());
+
+        Authentication auth = store.get();
+
         store.remove();
+
+        SecurityListener.fireLoggedOut(auth.getName());
```

Comment:
Makes [this Javadoc](https://github.com/jenkinsci/jenkins/blob/bcbd5ba56ad7bbed00977b2d733f60019c6af04c/core/src/main/java/jenkins/security/SecurityListener.java#L70) inaccurate.